dspace/config/spring/api/discovery.xml

```diff
@@ -894,6 +901,79 @@
         <property name="spellCheckEnabled" value="true"/>
     </bean>
 
+    <!--The other workspace configuration settings for discovery -->
+    <bean id="otherWorkspaceConfiguration"
```

Comment:
What do we mean by an "otherWorkspaceConfiguration"?  Could we change the name here (the phrase "other workspace" is not at all clear)? How does this relate to or differ from the "supervisionConfiguration?

Reply:
maybe supervisedWorkspaceConfiguration as there is already a supervionConfiguration used by the administrative dashboard?